sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1621,6 +1627,12 @@ public void onClick(DialogInterface dialog, int which) {
     @UiThread
     private synchronized void showUpdateSetupFailedDialog() {
 
+        /* Do not attempt to show dialog if application is in the background. */
```

Comment:
Did we test that resuming app restore the flow at the intended step?

Reply:
I see no reasons why it should not work. As a scope of the WI I've worked on NPEs on the first place, and If the behaviour is flawed (as in: not resumed), I'd better create a separate WI to address that.